js/core/renderer.d.ts

```diff
@@ -97,7 +97,7 @@ export interface dxElementWrapper {
 
   scrollLeft(value?: string | undefined): this;
 
-  scrollTop(value: string | undefined): this;
+  scrollTop(value?: string | undefined): this;
```

Comment:
Let's remove the ```undefined``` because we already use the ```?```

```suggestion
  scrollTop(value?: string): this;
```